openpype/hosts/max/plugins/publish/extract_thumbnail.py

```diff
@@ -0,0 +1,122 @@
+import os
+import tempfile
+import pyblish.api
+from openpype.pipeline import publish
+from pymxs import runtime as rt
+
+
+class ExtractThumbnail(publish.Extractor):
+    """
+    Extract Thumbnail for Review
+    """
+
+    order = pyblish.api.ExtractorOrder
+    label = "Extract Thumbnail"
+    hosts = ["max"]
+    families = ["review"]
+
+    def process(self, instance):
+        self.log.info("Extracting Thumbnail ...")
+
+        # TODO: Create temp directory for thumbnail
+        # - this is to avoid "override" of source file
+        tmp_staging = tempfile.mkdtemp(prefix="pyblish_tmp_")
+        self.log.debug(
+            f"Create temp directory {tmp_staging} for thumbnail"
+        )
+        instance.context.data["cleanupFullPaths"].append(tmp_staging)
+        filename = "{name}..jpg".format(**instance.data)
+        filepath = os.path.join(tmp_staging, filename)
+        filepath = filepath.replace("\\", "/")
+        thumbnail = self.get_filename(instance.name)
+
+        self.log.info(
+            "Writing Thumbnail to"
+            " '%s' to '%s'" % (filename, tmp_staging))
```

Comment:
This writes to a staging dir, the artist doesn't care. I'd say it's a debug log.
```suggestion
        self.log.debug(
            "Writing Thumbnail to"
            " '%s' to '%s'" % (filename, tmp_staging))
```